source/variables/custom-selectors.css

```diff
@@ -1 +1,8 @@
-@custom-selector :--enter :hover, :active;
\ No newline at end of file
+/* 
+	Allows you to create your own selectors
+	http://cssnext.io/features/#custom-selectors
+	:--enter {
+	  ... styles ...
+	}	
+*/
+@custom-selector :--enter :hover, :focus, :active;
```

Comment:
Well, I was asking why we were leaving it out, not insisting it needed to be added, and @krambuhl is effectively answering that question. Also, to be clear, when asking questions on a PR, I'm inviting conversation, not passively requesting immediate compliance.

That said, I'm still not clear, in speaking strictly about styling concerns, why or how often `:focus` and `:hover, :active` wouldn't be aligned. I particularly like them all together so that adding a11y-friendly `:focus` styles aren't an afterthought or forgotten altogether, and as `:hover` and `"active` styles _should_ have enough contrast and difference off the default that they should hypothetically work with `:focus` as well. Am I wrong about that? (To clarify these tone-deaf text boxes, I'm asking these questions and raising these points sincerely, want the discussion, and am happy to be swayed. Or, tl;dr: don't read this as any more aggressive or contrary than any other opinion. 😄)